src/Rector/Return_/CommandConstantReturnCodeRector.php

```diff
@@ -65,11 +64,11 @@ protected function execute(InputInterface $input, OutputInterface $output): int
      */
     public function getNodeTypes(): array
     {
-        return [ClassMethod::class];
+        return [Return_::class];
```

Comment:
The old ClassMethod is the correct way, as directly check ClassMethod name

Reply:
I think i resolved the merge incorrectly at some point thanks for finding it.